resources/scss/ceres/widgets/Header/_top-bar-widget.scss

```diff
@@ -216,3 +216,117 @@ html.ie { // stylelint-disable-line selector-no-qualifying-type
         min-height: 32px;
     }
 }
+
+// Currency & Country Select
+.cmp-country-settings,
+.cmp-currency-select {
+    position: fixed;
+    right: 0;
+    left: 0;
+    z-index: $zindex-7;
+    width: 100%;
+    background-color: $gray-900;
+
+    &.collapsing {
+        overflow: hidden;
+    }
+
+    .shipping-settings,
+    .language-settings {
+        ul {
+            @include media-breakpoint-down(sm) {
+                column-count: 2;
+            }
+        }
+    }
+
+    .shipping-settings {
+        @include media-breakpoint-down(md) {
+            margin-top: 1rem;
+        }
+    }
+
+    ul {
+        @include reset-list();
+        max-height: 50vh;
+        overflow-x: hidden;
+        overflow-y: auto;
+    }
+
+    li {
+        float: left;
+        width: 100%;
+
+        a {
+            display: inline-block;
+            padding: .4rem;
+            &:hover {
+                @include context-bg($primary);
+            }
+
+            & .is-disabled {
+                cursor: not-allowed;
+
+                &:hover {
+                    @include context-bg($gray-700);
+                }
+            }
+        }
+
+        & .active {
+            a {
+                @include context-bg($primary);
+            }
+            .is-disabled {
+                @include context-bg($gray-700);
+            }
+        }
+    }
+
+
+    @include media-breakpoint-up(md) {
+        ul {
+            li {
+                width: 50%;
+            }
+        }
+    }
+
+    @include media-breakpoint-up(lg) {
+        ul {
+            li {
+                width: 33.33%;
+            }
+        }
+    }
+
+    .currency-list {
+        li {
+            width: 16.666667%;
+        }
+    }
+}
+
+.flag-icon-en {
+    background-image: url("../images/flags/4x3/gb.svg");
+}
+
+.flag-icon-da {
+    background-image: url("../images/flags/4x3/dk.svg");
+}
+
+.flag-icon-nn {
+    background-image: url("../images/flags/4x3/no.svg");
+}
+
+.flag-icon-ic {
+    background-image: url("../images/flags/ic.svg");
+}
+
+.flag-icon-ea {
+    background-image: url("../images/flags/ea.svg");
+}
+
+.flag-icon-an {
+    background-image: url("../images/flags/an.svg");
+}
```

Comment:
Den Teil können wir in die icons.scss auslagern.

Reply:
done